### ISSUE TITLE
Add Pogo-Hopper counter

### ISSUE THREAD

**Issue by pszentgyorgyi** [HearthSim/Hearthstone-Deck-Tracker#3782]
It'd be lovely to know how big my/my opponents next pogo hopper will be.

**Comment by paradox7jcr** [comment]
This would be nice, but HS Deck tracker already tracks how many of each card your opponent has played.
So N pogo hoppers played means next pogo hopper has (n*2 + 2) attack and health. So the user SHOULD know how big the next pogo hopper will be just by looking at the cards opponent has played.

**Comment by pszentgyorgyi** [comment]
Hey!
I implemented the feature in September, it was merged back then. Interestingly enough the status of the PR is closed (not merged), but I see the changes in the history of master (commit 21740fdcaa74b2c20b19f277727be1b4455df3aa). I opened the ticket, so tracking the development is more transparent. It should've been closed when the PR was done. Feel free to close this ticket.

ps.: just for the sake of the argument: using the same logic, neither the Jade, nor the spell counters should have been implemented :)
not to mention that afaik it only tracks the cards up to 9.

Cheers!
